integration_tests/internal/main.go

```diff
@@ -39,7 +39,7 @@ func main() {
 func getEnvOrDie(name string) string {
 	val, ok := os.LookupEnv(name)
 	if !ok {
-		fmt.Fprintf(os.Stderr, name+" could not be found in environment")
+		fmt.Fprintf(os.Stderr, "%s could not be found in environment", name)
```

Comment:
Or this?

```suggestion
		fmt.Fprintln(os.Stderr, name+" could not be found in environment")
```

Reply:
As spaces are always added between operands, I just made it a bit simpler.

Reply:
Right, even better.